ydb/core/cms/console/console_configs_provider.h

```diff
@@ -100,14 +105,48 @@ class TConfigsProvider : public TActorBootstrapped<TConfigsProvider> {
         };
 
         struct TEvUpdateYamlConfig : public TEventLocal<TEvUpdateYamlConfig, EvUpdateYamlConfig> {
-            TEvUpdateYamlConfig(const TString &yamlConfig, const TMap<ui64, TString> &volatileYamlConfigs = {})
+            TEvUpdateYamlConfig(
+                const TString &yamlConfig,
+                const TMap<ui64, TString> &volatileYamlConfigs = {})
```

Comment:
Тут и далее:
```suggestion
                    const TString &yamlConfig,
                    const TMap<ui64, TString> &volatileYamlConfigs = {})
```
Иначе сливается со списком инициализации.